custom/_legacy/hsph/tasks.py

```diff
@@ -78,15 +78,11 @@ def get_group_id(domain, owner_type, facility_id):
     except KeyError:
         return None
 
-past_x_date = lambda time_zone, past_x_days: (datetime.datetime.now(time_zone) - datetime.timedelta(past_x_days)).date()
-get_none_or_value = lambda _object, _attribute: getattr(_object, _attribute) if (hasattr(_object, _attribute)) else ''
+past_x_date = lambda tz, past_x_days: (datetime.datetime.now(tz) - datetime.timedelta(past_x_days)).date()
+get_none_or_value = lambda _obj, _attr: getattr(_obj, _attr) if (hasattr(_obj, _attr)) else ''
 
 
-@periodic_task(
-    run_every=crontab(minute=1, hour="*/6"),
-    queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery')
-)
-def new_update_case_properties():
```

Comment:
Was there a reason this was moved to the bottom of the file? It makes the diff particularly hard to read.